rusk/Cargo.toml

```diff
@@ -21,13 +21,13 @@ path = "src/bin/main.rs"
 
 [dependencies]
 tonic = "0.3"
-tokio = { version = "0.2", features = ["rt-threaded", "time", "stream", "fs", "macros", "uds"] }
-dusk-poseidon = {version="0.20", features=["canon"]}
-dusk-plonk = { version = "0.7", default-features = false, features = ["canon"] }
+tokio = {version="0.2", features=["rt-threaded", "time", "stream", "fs", "macros", "uds"]}
+dusk-poseidon = {version="0.21.0-rc",features=["canon"]}
+dusk-plonk = {version="0.8", default-features=false, features=["canon"]}
 dusk-blindbid = "0.6"
-dusk-pki = "0.6"
-phoenix-core = {git="https://github.com/dusk-network/phoenix-core", tag = "v0.10.0"}
-schnorr = {git="https://github.com/dusk-network/schnorr", tag = "v0.6.0"}
+dusk-pki = "0.7.0-rc"
+phoenix-core = {git="https://github.com/dusk-network/phoenix-core", branch="release-0.11"}
```

Comment:
We should probably wait to have `phoenix-core` released.

Reply:
Fixed